libraries/Microsoft.Bot.Builder.Expressions/BuiltInFunctions.cs

```diff
@@ -738,7 +738,7 @@ private static (object value, string error) GetProperty(Expression expression, o
         /// <param name="instance">Instance with property.</param>
         /// <param name="property">Property to lookup.</param>
         /// <returns>Value and error information if any.</returns>
-        private static (object value, string error) AccessProperty(object instance, string property)
+        public static (object value, string error) AccessProperty(object instance, string property)
```

Comment:
Why did this need to become public?

Reply:
Because in the customziedMemoryScope i'm refering this to get the property